python/ml4ir/base/model/callbacks/debugging.py

```diff
@@ -29,6 +29,7 @@ def on_epoch_begin(self, epoch, logs=None):
         self.epoch = epoch + 1
         self.logger.info("Starting Epoch : {}".format(self.epoch))
         self.logger.info(logs)
+        self.logger.info('lr={}'.format(str(self.model.optimizer._decayed_lr(float).numpy())))
```

Comment:
@mohazahran Will `model.optimizer` have a `_decayed_lr` attribute when running with other LR settings?

Reply:
@mohazahran , waiting on your confirmation here. Based on tf code it seems to be the case

Reply:
it seems that way yes. I tried with both sgd and adam